pom.xml

```diff
@@ -872,6 +880,7 @@
     </pluginManagement>
   </build>
   <reporting>
+    <outputDirectory>${basedir}/target/site</outputDirectory>
```

Comment:
The output was in `${basedir}/target` before, right?

Reply:
The current output is in `${basedir}/target/artifacts/site`.

Reply:
i am not sure if we want to move this outside the artifacts directory @SaltyAimbOtter what do you mean?

Reply:
We will do this, so i resolve this